R/matrix_form.R

```diff
@@ -894,6 +894,7 @@ mf_has_rlabels <- function(mf) ncol(mf$strings) > mf_ncol(mf)
 #' mform <- basic_matrix_form(mtcars)
 #' cat(toString(mform))
 #'
+#' @examplesIf require("dplyr")
```

Comment:
I checked and this is the only place! I think we can also move towards doing the same in {rtables}. What do you think @pawelru @shajoezhu?

Reply:
I think there is no other choice to pass noSuggest check